web/client/plugins/TOC.jsx

```diff
@@ -469,6 +475,29 @@ class LayerTree extends React.Component {
     }
 }
 
+const securityEnhancer = (Component) => (props) => {
+    const { addLayersPermissions = true,
+        removeLayersPermissions = true,
+        sortingPermissions = true,
+        addGroupsPermissions = true,
+        removeGroupsPermissions = true, user, ...other} = props;
+
+    const activateParameter = (allow, activate) => {
+        const isUserAdmin = user && user.role === 'ADMIN' || false;
+        return allow ? activate : isUserAdmin ? activate : false;
```

Comment:
```suggestion
        return (allow || isUserAdmin) ? activate : false;
```
I think is more clear rewritten like this.